data/tribes/workers/amazons/woodcutter/init.lua

```diff
@@ -19,6 +19,11 @@ wl.Descriptions():new_worker_type {
 
    programs = {
       harvest = {
+         -- steps from building to tree: 2-13, min+max average 7.5
+         -- min. worker time: 2 * 2 * 1.8 + 30 + 2 = 39.2 sec
+         -- max. worker time: 2 * 13 * 1.8 + 30 + 2 = 78.8 sec
+         -- avg. worker time: 2 * 7.5 * 1.8 + 30 + 2 = 59 sec
+         -- some of trees delay woodcutting by 1.4 sec (aspen, oak)
```

Comment:
this should probably be fixed for all trees to have the same cycle values 
(i.e. wee need to include a delay of 1.4 for all of them (until we may define proper falling animations) and turn all trees into fallen trees), but not in this PR thanks for finding this I'll raise an issue